documents/TestSuite/_options.mtlx

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Test suite options file. 
+-->
+<materialx version="1.36">
+    <nodedef name="ShaderValidOptions">
```

Comment:
About the naming of the options nodedef. If these are general test options maybe name the nodedef TestOptions instead.
If the idea is to have support for multiple of these for particular tests then maybe create a nodedef for the definition of the general options struct, and then support having "instances" of this for different tests:

```
<nodedef name="TestOptions">
    <parameter name="foo" type="boolean" value="false" />
    <parameter name="bar" type="boolean" value="true" />
</nodedef>

<TestOptions name="ShaderValid">
    <parameter name="foo" type="boolean" value="true" />
    <parameter name="bar" type="boolean" value="true" />
</TestOptions>

<TestOptions name="SomeOtherTest">
    <parameter name="foo" type="boolean" value="true" />
    <parameter name="bar" type="boolean" value="true" />
</TestOptions>
```
(No sure there is a need for this but just an idea).

Reply:
Currently its very specific to the ShaderValid test but will probably rename this and the C++ structure to "ShaderValidTestOptions".

Reply:
I think this would be a good idea for output options like image size, format and maybe for lighting options(?). So:
```
<nodedef name="RenderingOptions">
<parameter name="imageWidth" type="integer" value="256" />
<parameter name="imageHeight" type="integer" value="256" />
<parameter name="channelCount" type="integer" value="4" />
<parameter name="channelFormat" type="string" value="float" />
<parameter name="imageFormat" type="string" value="png" />
</nodedef>
```
Can maybe take a look later to see what is useful here.